### ISSUE TITLE
support es6 in external js files

### ISSUE THREAD

**Issue by rosshadden** [ConnectAi/cs#13]
This will be easy now that we moved to `6to5`.

We'll just transpile things in `private/js/` to `public/js/`.

It should be togglable in the config.

**Comment by optikalefx** [comment]
it should say compiler: {}. someone might want...... COFFEESCRIPT.

**Comment by rosshadden** [comment]
Yeah..... guess we have to handle _those_ kinds of people.  And typescript, etc.  Obviously(?) these would be cs-* modules.  Even `6to5` could/should be it's own module I suppose.

**Comment by optikalefx** [comment]
Maybe, we should just look at what we have to do to compile the private folder. I suppose they would be cs-* modules.  it should compile into the same dir structure in public for easy debugging.  And then a later grunt file could expect and concat / minify that code.  Even if it's coffeescript it should do the same thing. So maybe all we have to do is pass a compile function and handle the rest? If that's the case, then we don't need a cs-* we just need to know what the compile function is and use it.
